chapter_multilayer-perceptrons/mlp_vn.md

```diff
@@ -368,7 +369,7 @@ Given the element $z$, the function is defined as the maximum of that element an
 
```

Comment:
L350: bỏ chữ `nó` cuối cùng.

L351:  `Chúng là các toán tử khả vi có vai trò biến đổi các tín hiệu đầu vào thành các tín hiệu đầu ra, và hầu hết các hàm kích hoạt này đều thêm sự phi tuyến tính.`
--> `Chúng là các toán tử khả vi và hầu hết đều biến đổi các tín hiệu đầu vào thành các tín hiệu đầu ra theo một cách phi tuyến tính.`